CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,79 @@
+# Code of Conduct
+
+In order to foster a welcoming, positive, and rewarding experience, we as
+community members pledge to uphold this code of conduct.
+
+*Isn't decent behavior is already expected?* We want to bring our best to each
+other as peers, and explicitly documenting our expectations lets us set a high
+bar for ourselves.
+
+## Be excellent to each other
+
+"Excellent" means we:
+
+- **are wholly inclusive** to existing members and newcomers looking to
+  learn or participate. Some may raise questions that have been addressed
+  before. Kindly point them to existing discussions. Everyone was new to a
+  project once.
+- **respect** your abilities, interests, and personal differences.  Our
+  different talents documenting, testing, coding, reviewing, reporting, and/or
+  evangelizing are all valuable to the project.
+- **make concise and actionable points** to respect your time as a volunteer.
+  We respect each other's time by keeping the signal-to-noise ratio high,
+  staying on-topic, and being a good house-guest: every visit we strive to
+  leave the place in better shape than when we arrived.
+- **are kind** and **assume positive intent**, even when we disagree while
+  working towards consensus and achieving the project's technical goals.
+- **educate and illuminate** others with something we know more about. We were
+  all beginners once, and can teach and learn through feedback and sharing.
+- **are open to feedback** from others, be it technical points, interpersonal
+  tips, or language/wording suggestions.
+- **accept responsibility** by apologizing to those affected by our mistakes
+  and graciously accepting apologies knowing that mistakes are part of the
+  process.
+- **use words carefully**, because we may not understand irony or negative
+  sarcasm. We communicate plainly, and we justify our positions objectively and
+  explicitly.
+- **leave with class**, without drama or bitterness, when resigning for any
+  reason. We champion your rights under the terms of the GPL to fork the code
+  and create a new project, just as we stand on the shoulders of those who gave
+  us the right to create this project.
+
+## Reporting Issues
+
+Please bring up issues by contacting
+[dreamer](mailto:patryk.obara+coc@gmail.com) or
+[krcroft](mailto:k.r.croft+coc@gmail.com) who will handle your report with
+discretion. Your report should include:
+
+- Names (real or usernames) of those involved in the incident.
+- Your description of what occurred, and if the incident is ongoing. Please
+  provide links to or transcripts of the publicly available records (e.g. a
+  mailing list archive, GitHub issue, public IRC log, or Discord transcript),
+  so that we can review it.
```

Comment:
>Please provide links or transcripts of the publicly available records (e.g. a mailing list archive, GitHub issue, public IRC log, or Discord transcript), so that we can review the evidence. Private communication is outside our scope, we suggest utilizing the blocking features on the platform in question.

Reply:
I believe policing private communications to be impossible, so I added a clause to reflect that.

I hope folks remain as civil in private as they are in public, but every platform has a blocking feature so the user has enough power here to solve the problem themselves.

Reply:
Nicely said, and a great recommendation regarding the blocking feature. 
Added.